lessons/3_first_interactive_component/index.html

```diff
@@ -331,29 +364,92 @@ <h3>For that you need to do:</h3>
 					</li>
 				</ul>
 			</section>
+			<section>
+				<h3>But where do we write JavaScipt code?</h3>
+			</section>
+			<section>
+				<p>We will use <b>&lt;script/&gt;</b> tag inside <b>index.html</b></p>
+				<pre>
+					<code>
+&lt;html&gt;
```

Comment:
Christ. did you have to hand type this or did you use encode tool?

Reply:
nope, I did copy most of it stuff from other slides :)